classes/class-uagb-post-assets.php

```diff
@@ -258,20 +258,36 @@ public function __construct( $post_id ) {
 	public function get_woocommerce_template() {
 		// Check if WooCommerce is active.
 		if ( class_exists( 'WooCommerce' ) ) {
+			// Check other WooCommerce pages.
 			if ( is_cart() ) {
-				return 'cart';
+				return 'page-cart';
 			} elseif ( is_checkout() ) {
-				return 'checkout';
-			} elseif ( is_shop() ) {
-				return 'archive-product';
+				return 'page-checkout';
 			} elseif ( is_product() ) {
 				return 'single-product';
-			} elseif ( is_product_taxonomy() ) {
-				return 'taxonomy-product_cat';
-			} elseif ( is_product_tag() ) {
-				return 'taxonomy-product_tag';
-			} elseif ( is_product_category() ) {
-				return 'taxonomy-product_cat';
+			} elseif ( is_archive() ) { // Check if the current page is an archive.
+				$object = get_queried_object();
+				// Check if it's a search on the product archive.
+				$searchCondition = is_search() && is_post_type_archive( 'product' );
```

Comment:
Since we're using `product-search-results` wherever the search condition is true, let's keep it as the value of the search condition and then use the Elvis operator to return.

```suggestion
				$product_search_template = ( is_search() && is_post_type_archive( 'product' ) ) ? 'product-search-results' : '';
```

Then down the line like this:
```
return $product_search_template ?: 'taxonomy-product_attribute';
```

Reply:
@bsf-zanev Returning `$product_search_template ?: 'taxonomy-product_attribute';` like this is not allowed in WordPress PHP. Getting PHPCS error (Using short ternaries is not allowed) after implementation.